src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -635,6 +637,70 @@ class spell_dk_glyph_of_scourge_strike_script : public SpellScript
     }
 };
 
+static constexpr uint32 SPELL_VISUAL_ID_HOWLING_BLAST = 66812;
+
+// 49184 - Howling Blast
+// 237680 - Howling Blast
+class spell_dk_howling_blast : public SpellScript
+{
+    PrepareSpellScript(spell_dk_howling_blast);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DK_HOWLING_BLAST_AREA_DAMAGE, SPELL_DK_FROST_FEVER });
+    }
+
+    void FilterTargets(std::list<WorldObject*>& targets)
+    {
+        targets.remove_if([&](WorldObject const* target)
+        {
+            if (GetSpellInfo()->Id != SPELL_DK_HOWLING_BLAST_AREA_DAMAGE)
```

Comment:
we literally just did it like this.. there was no need for an extra target list or smth
![grafik](https://user-images.githubusercontent.com/639817/194181943-113e504e-c7d2-4e40-84cf-fcdb62553b10.png) 
![grafik](https://user-images.githubusercontent.com/639817/194181895-cea6a8ba-efaa-4faa-aae0-570ddcf13466.png)

Reply:
If you‘re using a unit to cast it, even if it’s a dest spell, it can still hold a unit. Check the code you will see that it can hold both.

Reply:
> If you‘re using a unit to cast it, even if it’s a dest spell, it can still hold a unit. Check the code you will see that it can hold both.

Ok actually i gotta recheck why it did work back in legion.. it had the same effects and target types but i honestly dont understand atm why it worked. We also didn't do any custom changes.